spark/src/main/scala/geotrellis/spark/stitch/StitchRDDMethods.scala

```diff
@@ -41,6 +41,58 @@ abstract class SpatialTileLayoutRDDStitchMethods[
     val (ulx, uly) = (base.x - offsx.toDouble * layout.cellwidth, base.y + offsy * layout.cellheight)
     Raster(tile, Extent(ulx, uly - tile.rows * layout.cellheight, ulx + tile.cols * layout.cellwidth, uly))
   }
+
+  /**
+    * Stitch all tiles in the RDD using a sparse stitch that handles missing keys
+    *
+    * Any missing tiles within the extent are filled with an empty prototype tile.
+    *
+    * @note This method performs an RDD.collect() so ensure your dataset fits
+    *       into driver memory.
+    *
+    * @param extent The requested [[geotrellis.vector.Extent]] of the output [[geotrellis.raster.Raster]]
+    * @return The stitched Raster, otherwise None if the collection is empty or the extent does not intersect
+    */
+  def sparseStitch(extent: Extent): Option[Raster[V]] = {
+    val tiles = self.toCollection
+    // From here down, this code duplicates SpatialTileLayoutCollectionStitchMethods.sparseStitch,
+    // replacing self with tiles
+    if (tiles.headOption.isEmpty) {
+      None
+    } else {
+      val tile = tiles.head._2
+      val layoutDefinition = self.metadata.getComponent[LayoutDefinition]
+      val mapTransform = layoutDefinition.mapTransform
+      val expectedKeys = mapTransform(extent)
+        .coordsIter
+        .map { case (x, y) => SpatialKey(x, y) }
+        .toList
+      val actualKeys = tiles.map(_._1)
+      val missingKeys = expectedKeys diff actualKeys
+
+      val missingTiles = missingKeys.map { key =>
+        (key, tile.prototype(layoutDefinition.tileLayout.tileCols, layoutDefinition.tileLayout.tileRows))
+      }
+      val allTiles = tiles.withContext { collection =>
+        collection.toList ::: missingTiles
```

Comment:
I prefer the `++` operator for concatenation. It works with non-lists and does the same thing as `:::` (falling back to it when dealing with lists according to a SO post I recall having read)

Reply:
I don't feel strongly about it, so I updated it. Thanks for taking a look at the PR.